test/cases/testdata/array/test-array-0052.yaml

```diff
@@ -0,0 +1,52 @@
+cases:
+- note: 'array/reverse_123'
+  query: data.generated.p = x
+  data:
+    foo:
+    - 1
+    - 2
+    - 3
+  modules:
+  - |
+    package generated
+
+    p = x {
+      array.reverse(data.foo, __local0__)
```

Comment:
Do we need `__local0__` here? 

Would not
```rego
p = x {
      x := array.reverse(data.foo)
}
```
or even

```rego
p := array.reverse(data.foo)
```
suffice?

Reply:
Totally. Changed to that now. Thanks 😄